pkg/validation/policy/validate.go

```diff
@@ -12,7 +12,6 @@ import (
 
 	"github.com/distribution/distribution/reference"
 	jsonpatch "github.com/evanphx/json-patch/v5"
-	"github.com/jmespath/go-jmespath"
```

Comment:
Seems to me like this import removal breaks the build.

Reply:
Fixed the missing import cc @Dentrax